index.html

```diff
@@ -395,6 +395,27 @@ <h2>Convert a <var>name</var> to a <var>timestamp</var></h2>
         The <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface was defined in [[NAVIGATION-TIMING]] and is now considered obsolete. The use of names from the <a data-cite="NAVIGATION-TIMING#performancetiming">PerformanceTiming</a> interface is supported to remain backwards compatible, but there are no plans to extend this functionality to names in the <a data-cite="NAVIGATION-TIMING-2#dom-performancenavigationtiming">PerformanceNavigationTiming</a> interface defined in [[NAVIGATION-TIMING-2]] (or other interfaces) in the future.
       </p>
     </section>
+  </section><section id="recommended-mark-names">
+    <h2><dfn>Recommended mark names</dfn></h2>
```

Comment:
Unfortunately, ReSpec doesn't yet support data-export on headers, so <dfn class="export"> is good: 

```suggestion
    <h2><dfn class="export">Recommended mark names</dfn></h2>
```